drivers/sony/displays/bravia.cr

```diff
@@ -169,21 +168,12 @@ class Sony::Displays::Bravia < PlaceOS::Driver
   }
   RESPONSES = COMMANDS.to_h.invert
 
-  TYPES = {
-    control: "\x43",
-    enquiry: "\x45",
-    answer:  "\x41",
-    notify:  "\x4E",
-  }
-
-  BINARY_TYPE = {
-    65 => "A",
-    69 => "E",
-    67 => "C",
-    78 => "N",
-  }
-
-  TYPE_RESPONSE = TYPES.to_h.invert
+  enum MessageType : UInt8
+    Answer  = 0x41
+    Control = 0x43
+    Enquiry = 0x45
+    Notify  = 0x4e
```

Comment:
```suggestion
    Notify  = 0x4e
    
    def control_character
      case self
      in Answer  then "A"
      in Control then "C"
      in Enquiry then "E"
      in Notify  then "N"
      end
    end
```

Reply:
Those values are the characters. If you want them directly [`Int#chr`](https://crystal-lang.org/api/1.0.0/Int.html#chr-instance-method) is available. This works well if you are serialising elsewhere (e.g. when building a command packet before transmission).
```crystal
MessageType::Answer.value.chr # => 'A'
```

If the primary serialisation is the protocol form (e.g. these are not appear in a bunch of log messages or similar) a nice pattern for this though to pop a `#to_s(IO)` override straight on the enum:
```crystal
enum MessageType : UInt8
  Answer  = 0x41
  Control = 0x43
  Enquiry = 0x45
  Notify  = 0x4e

  def to_s(io)
    io << self.value.chr
  end
end

String.build { |io| io << MessageType::Answer }  # => "A"

"Foo#{MessageType::Answer}" # => "FooA"